src/components/DocumentWidget.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <router-link
+    tag="div"
+    v-if="document.id && partnerId"
+    path="Dados do documento"
+    :to="{ name: 'empresas.documentos.view', params: {partnerId: partnerId, documentId: document.id } }"
+    class="DocumentWidget d-flex flex-column justify-content-start align-items-start text-left"
+  >
+    <!-- Lorempixel down! -->
+    <div class="--Header d-flex flex-row">
+      <font-awesome-icon icon="stamp" :style="document.vencido ? {color: '#14CB93'} : {color: '#FFB11B'}"/>
+      <div class="--Expired" :style="document.vencido ? [{backgroundColor: '#14CB93'}] : [{backgroundColor: '#FFB11B'}]">
+        {{document.vencido ? 'Válido' : 'Inválido'}}
+      </div>
+    </div>
+    <span class="--Name">{{document.nome}}</span>
+    <span>Criado em {{formatedCreatedAt}}</span>
+  </router-link>
+</template>
+
+<script>
+import moment from 'moment';
+export default {
+  name: "DocumentWidget",
+  props: {
+    partnerId: String,
+    document: Object|| null
+  },
+  computed: {
+    formatedCreatedAt: function () {
+      return (document ? moment(document.criadoEm).format('DD/MM/YYYY') : null);
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+$screen-sm-max: 578px;
+.DocumentWidget {
```

Comment:
Qual é a referência para esse padrão de CSS usando letras em Camel Case -- ou __ no início?